CHARTER.md

```diff
@@ -0,0 +1,108 @@
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. The JSON Schema project aims to enable these additional and emergent use cases.
+
+### 1.1: In-scope
+The scope of the JSON Schema project is split into two sections: primary and secondary concerns.
+Primary concerns are areas the JSON Schema project wish to give focus to. Secondary concerns, while remaining in scope, aren't areas of focus, and would require dedicated championing by community members to come to fruition.
+
+Primary Concerns
+- Publication of the JSON Schema standard
+  - Validation of JSON-compatible data
+  - Semantic annotation of JSON-compatible data
+  - Interoperability
+  - Extensibility
+- Critical tooling
+- Documentation
+- Test suite
+- Community
+  - Enabling schema authors
+  - Enabling implementers
+  - Engaging with industry
+  - Engaging with upstream and downstream standards and projects
+  - Communicating value
+  - Ensuring the sustainability of the project
+
+Secondary Concerns
+- Hypermedia
+- Generating JSON Schema
+- Using JSON Schema to generate
+  - Code (including types or classes)
+  - UI (including forms)
+  - Databases
+- Relational validation
+- Vocabularies registry
+
+### 1.2: Out-of-Scope
+Neither standards that the JSON Schema project uses (such as JSON and IRI) nor standards or projects that use JSON Schema (such as OpenAPI or AsyncAPI) are in scope, nor does the JSON Schema project have any control over them.
+
+## 2: Relationship with OpenJS Foundation CPC.
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of the JSON Schema project, it is delegated to the JSON Schema Technical Steering Committee ("TSC").
+
+### 2.1 Other Formal Project Relationships
+Section Intentionally Left Blank
+
+## 3: JSON Schema Governing Body (TSC)
+The JSON Schema Technical Steering Committee (TSC) is initially established from the observed major contributors who are currently active and in good standing.
+
+The TSC must have a minimum of four members. There is no maximum TSC membership size.
+
+TSC memberships are not time-limited.
+
+The TSC follows the decision-making process defined in this charter unless otherwise documented.
+
+The TSC aims to work asynchronously. TSC meetings are pre-announced, public, and recorded. Minutes are taken and the recordings made available. If there is a reasonable reason (such as security or privacy), any portion of a meeting, its minutes, and recording, may be kept private.
+
```

Comment:
Any restrictions on the maximum percentage of Governing Body members that can be employed by the same company? The risk of one company exerting undue influence over the direction of the project is real.

Reply:
We looked at this before, I but I don't remember (or necessarily agree with) why it was removed.